mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/HeaderCollectionTest.java

```diff
@@ -62,7 +62,16 @@ public void displayNamesShouldBeRetreived() {
     }
 
     @Test
-    public void addressWithTwoDisplayNamesOnTheSameFieldShouldBeRetreived() {
+    public void displayNamesShouldBeRetrievedWhenEncodedWord() {
+        HeaderCollection headerCollection = HeaderCollection.builder().add(new FieldImpl("From", "=?UTF-8?B?RnLDqWTDqXJpYyBNQVJUSU4=?= <fmartin@linagora.com>, Graham CROSMARIE <gcrosmarie@linagora.com>")).build();
+        assertThat(headerCollection.getFromAddressSet())
+            .containsOnly(new EMailer("Frédéric MARTIN", "fmartin@linagora.com"),
```

Comment:
you don't really care for that : you just want to make sure "Frédéric" is ok, so you can do `.extract(Emailer::name).contains("Frédéric MARTIN")` (it probably doesn't compile, but I guess you got the point).